Simplenote/src/main/java/com/automattic/simplenote/authentication/MagicLinkableFragment.kt

```diff
@@ -21,9 +21,10 @@ import com.simperium.android.ProgressDialogFragment
  */
 abstract class MagicLinkableFragment : Fragment() {
```

Comment:
You are not wrong. Let me table this one. I don't want to make this update now because once I merge trunk, I will get a ton of merge conflicts. I can make the update in a separate PR if you don't mind.

Reply:
Actually, I ended up doing it. Didn't realize it is currently only being used by `SignInFragment`. The goal is to also use it for signup. So name it `BaseAuthenticationFragment`.